incubator/hnc/test/e2e/issues_test.go

```diff
@@ -194,6 +194,8 @@ var _ = Describe("Issues", func() {
 		mustRun("kubectl hns create", nsSubChild, "-n", nsChild)
 		// Test: remove non-leaf subnamespace with 'allowCascadingDelete' unset.
 		// Expected: forbidden because 'allowCascadingDelete'flag is not set
+		// wait 2 seconds for the sub-child namespace to be created 
+		time.Sleep(2*time.Second)
```

Comment:
Sleeps are usually the wrong way to solve a problem - if we're waiting for the namespace to be created, can you make that explicit?

Reply:
done